test-data/unit/check-generics.test

```diff
@@ -648,6 +648,28 @@ reveal_type(f3()) # N: Revealed type is "Union[builtins.int, __main__.Node[built
 
 [builtins fixtures/list.pyi]
 
+[case testGenericTypeAliasesWithNestedArgs]
+import other
+a: other.Array[float]
+reveal_type(a) # N: Revealed type is "other.array[Any, other.dtype[builtins.float]]"
+
+[out]
+main:2: error: Type argument "float" of "dtype" must be a subtype of "generic"
```

Comment:
output with the current master is
```
  main:12: error: Type argument "float" of "dtype" must be a subtype of "generic" (diff)
```
where 12 is the line number in other.py but is used as a reference with main.py.

Reply:
with `--pretty` flag the test fails on master as
```
[gw0] linux -- Python 3.8.10 /home/miha/foss/mypy/venv/bin/python3
data: /home/miha/foss/mypy/test-data/unit/check-generics.test:651:
/home/miha/foss/mypy/mypy/test/testcheck.py:141: in run_case
    self.run_case_once(testcase)
/home/miha/foss/mypy/mypy/test/testcheck.py:199: in run_case_once
    res = build.build(sources=sources,
/home/miha/foss/mypy/mypy/build.py:181: in build
    result = _build(
/home/miha/foss/mypy/mypy/build.py:257: in _build
    graph = dispatch(sources, manager, stdout)
/home/miha/foss/mypy/mypy/build.py:2752: in dispatch
    process_graph(graph, manager)
/home/miha/foss/mypy/mypy/build.py:3110: in process_graph
    process_stale_scc(graph, scc, manager)
/home/miha/foss/mypy/mypy/build.py:3227: in process_stale_scc
    manager.flush_errors(manager.errors.file_messages(graph[id].xpath), False)
/home/miha/foss/mypy/mypy/errors.py:707: in file_messages
    return self.format_messages(self.error_info_map[path], source_lines)
/home/miha/foss/mypy/mypy/errors.py:680: in format_messages
    source_line = source_lines[line - 1]
E   IndexError: list index out of range
```